torch_geometric/nn/dense/mincut_pool.py

```diff
@@ -1,14 +1,12 @@
-from typing import Optional, Tuple
+from typing import Optional, Tuple, Union
 
 import torch
 from torch import Tensor
 
 
 def dense_mincut_pool(
-    x: Tensor,
-    adj: Tensor,
-    s: Tensor,
-    mask: Optional[Tensor] = None,
+    x: Tensor, adj: Tensor, s: Tensor, mask: Optional[Tensor] = None,
+    temp: Optional[Union[float, int]] = 1.0
```

Comment:
```suggestion
    temp: float = 1.0
```
+ `temp` cannot be `None`
+ `temp` is better to be `float`

Reply:
I have updated as per your suggestion.